src/bin/default.js

```diff
@@ -1,6 +1,12 @@
 import path from 'path';
 import {isCI} from '../lib/ci';
 
+const chromedriver = require('../../../../package').devDependencies.chromedriver;
+
+if (!chromedriver) {
+  throw new Error('Please install chromedriver as a dev dependency in your repo!')
```

Comment:
not repo, project/package. Would be good to add instructions or link about how to do so.